package.json

```diff
@@ -32,8 +32,8 @@
     "format": "prettier --write '**/*.{yml,json,md,html,ts,js}'",
     "format:check": "prettier --check '**/*.{yml,json,md,html,ts,js}'",
     "prod": "npm run compile && npm run bundle && npm run launch",
-    "compile": "npm run copy-assets && tsc",
-    "compile:watch": "npm run copy-assets && tsc -w",
+    "compile": "npm run copy-assets && tsc --project tsconfig.build.json",
```

Comment:
what does this do, prevent linting on the mocks?

Reply:
no it enables linting the mocks - typescript-eslint needs things to be in the tsconfig to lint them.  So we have a root tsconfig where everything is enabled and then exclude mocks when building in this tsconfig